package.json

```diff
@@ -87,6 +87,6 @@
     "typescript-tslint-plugin": "0.5.3"
   },
   "engines": {
-    "node": "12"
+    "node": "^14||^16"
```

Comment:
Do we need to specify an npm version here so that the lockfile version is compatible? Not sure with yarn tbh...

Reply:
I think this is ok? I couldn't find much on specifying one without the other. Most are around just the node version. If we encounter problems we can add it in later on.